src/cata_tiles.cpp

```diff
@@ -645,6 +645,11 @@ void tileset_loader::load( const std::string &tileset_id, const bool precheck )
             for( const JsonObject mod_config : mod_config_json.get_array() ) {
                 if( mod_config.get_string( "type" ) == "mod_tileset" ) {
                     if( num_in_file == mts.num_in_file() ) {
+                        // visit this if it exists, it's used elsewhere
+                        if( mod_config.has_member( "compatibility" ) ) {
+                            std::vector<std::string> tmp;
+                            mod_config.read( "compatibility", tmp );
```

Comment:
```suggestion
                            mod_config.get_member( "compatibility" );
```
I think this reads and marks the member as visited without actually deserializing it, which should be somewhat faster.